test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -15,6 +15,7 @@ check_result_qemu() {
     umount $TESTDIR/root
     [[ -f $TESTDIR/failed ]] && cat $TESTDIR/failed
     ls -l $TESTDIR/journal/*/*.journal
+    journalctl -u testsuite --no-pager --file=$TESTDIR/journal/*/*.journal
```

Comment:
There is https://anonscm.debian.org/cgit/pkg-systemd/systemd.git/commit/debian/tests/upstream?id=063876a7297ef0ae17ccc697667b0cce0e9e4880
i.e. we don't need to print logs here.
But I think that's a good idea to move `journalctl` to the `check_result_qemu`.
@martinpitt , what do you think?
(Anyway, I'd prefer to remove this for now)